src/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "solr_url": "https://mars.cyverse.org/thing/select",
+  "solr_url": "https://hyde.cyverse.org/isamples_central/thing/select",
```

Comment:
We should revert this change -- though Mars was down while I was working on it -- Mars is our dedicated dev server and we should default to pointing there.